README.md

```diff
@@ -2,3 +2,16 @@
 [![Build Status](https://travis-ci.org/SMaxOwok/ui-pictionary.svg?branch=master)](https://travis-ci.org/SMaxOwok/ui-pictionary)
 
 ![picationary](static/pictionary.gif)
+
+# Setup
+Requirements: Ruby version 2.5.7, Bundler 2
+- Install nvm and yarn
+  - You may need to `yarn cache clean` and then `yarn install`
+- The local webpack server is run with `ruby bin/webpack-dev-server`
+- UI-Pictionary uses Sidekiq for background processing, and Sidekiq requires Redis
+  - Install and run redis `brew install redis` then `brew services start redis`
+
+clock: ruby bin/clockwork ./clock.rb
```

Comment:
We can leave this here with a note about running processes separately in different tabs.  We shouldn't need the `ruby` prefixes on them anymore either.